### PR TITLE
Cluster manager health v2

### LINKED ISSUE #5633: Add health check for Storage/Cluster management module
In the previous releases, @hanahmily added the health check module and its implementation for the storage modules(H2, MySQL, and ES), we need to add a similar thing for InfluxDB.

Also, for cluster management, we need to add this too. If 
1. the registration fails 
1. can't get the instance list 
1. can't get itself, then
1. get multiple itself(such as multiple 127.0.0.1/0.0.0.0) in the instance list
The cluster management should be listed as an error 

The health check could be verified through CLI command, https://github.com/apache/skywalking-cli#checkhealth

@xbkaishui Are you going to do this?

### PR DESCRIPTION
**Pull request by xbkaishui**
- [x] Closes #5633 add health check for cluster manager
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/c2141978d1039375598a32418b75161a78da22c1/CHANGES.md).

### CODE REVIEW

File: oap-server/server-cluster-plugin/cluster-zookeeper-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/zookeeper/ZookeeperCoordinator.java [hunk 1]
```diff
@@ -22,29 +22,40 @@
...
 
     private static final String REMOTE_NAME_PATH = "remote";
 
     private final ClusterModuleZookeeperConfig config;
     private final ServiceDiscovery<RemoteInstance> serviceDiscovery;
     private final ServiceCache<RemoteInstance> serviceCache;
     private volatile Address selfAddress;
+    @Setter
+    private HealthCheckMetrics healthChecker;
+    private final ModuleDefineHolder manager;
```
Comment (wu-sheng) [inline oap-server/server-cluster-plugin/cluster-zookeeper-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/zookeeper/ZookeeperCoordinator.java:54]: Move final to following other `final` fields in the top of the list.

File: oap-server/server-cluster-plugin/cluster-zookeeper-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/zookeeper/ZookeeperCoordinator.java [hunk 2]
```diff
@@ -22,29 +22,40 @@
...
 public class ZookeeperCoordinator implements ClusterRegister, ClusterNodesQuery {
-    private static final Logger LOGGER = LoggerFactory.getLogger(ZookeeperCoordinator.class);
 
     private static final String REMOTE_NAME_PATH = "remote";
 
     private final ClusterModuleZookeeperConfig config;
     private final ServiceDiscovery<RemoteInstance> serviceDiscovery;
     private final ServiceCache<RemoteInstance> serviceCache;
     private volatile Address selfAddress;
+    @Setter
```
Comment (wu-sheng) [inline oap-server/server-cluster-plugin/cluster-zookeeper-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/zookeeper/ZookeeperCoordinator.java:52]: You don't need the setter I think. Please don't open unnecessary access.

**Review by wu-sheng — COMMENTED** [review]
Two code style issues, please fix them in all implementations. Others look good to me.

**Comment by xbkaishui** [comment]
Greate, will fix now

**Comment by xbkaishui** [comment]
Done

**Review by wu-sheng — APPROVED** [review]
LGTM

**Pull request merged**